### DISCUSSION TITLE
What is this highlight pop-up called and how is it configured?

### DISCUSSION THREAD

**Discussion by gandhiShepard** [nvim-treesitter/nvim-treesitter#3597] [Q&A]
![Screen Shot 2022-10-06 at 4 53 52 PM](https://user-images.githubusercontent.com/8546059/194426099-69bbe7cc-5efa-483e-9636-af3ccfe3555a.jpg)

When running `:TSHighlightCapturesUnderCursor` a pop-up appears and displays the highlight group like:
# Treesitter
@comment

It used to display the highlight info somthing like:
"comment > htmlTSComment"

I'm asking this because I have a bunch of highlight groups defined in my theme using the old labeling. I'm targeting `htmlTSComment` however, should I be targeting these differently now? Ultimately I'm looking to maintain language based styling. For example a class in CSS is not the same as a class in JS. With the new display from `:TSHighlightCapturesUnderCursor` it's not obvious how to maintain language based highlight groups. 

I'm sorry if I'm missing a detail somewhere. Please advise. MUCH RESPECT!!!

**Comment by theHamsta** [comment]
It comes from nvim-treesitter/playground and can not be configured at the moment. It should have it's separate implementation file if you want to create a custom modification of it.

**Reply by gandhiShepard** [reply]
ok, gotcha! Thanks.

**Comment by clason** [comment]
Language specific highlight groups always work in the same way: adding `.language` at the end. So in your case `@comment.html`.
